perfs_parallel_computing.rst

```diff
@@ -166,31 +281,44 @@ process to another in order to ensure they all have a similar load. Activate it
 in the input file using::
 
     LoadBalancing(
-        every = 100
+        every = 20
     )
 
-Then run the simulation again with 12 processes.
+Then run the simulation again with 16 processes and have a look to the ``Load balancing`` timer. 
+You can observe differences in the computation time and
+compare it to the time saved regarding the simulation without dynamic load balancing.
+
+
+.. warning::
+
+  ``Sync`` timers are impacted by the imbalance of the algorithm part which precedes it :
+  
+   * ``Particles``
+   * ``Sync Densities``
+   * ``Maxwell``
+   * ``Sync Particles``
+   * ``Sync Fields``
 
-How are the regions modified? Can you observe differences in the computation time?
+
+Have look to the performance diagnostic and especially to the regions distribution.
 
 
 ----
 
-Balancing the load inside one process
+Real life configuration
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-Now, instead of splitting the simulation between 12 processes, let us
-use only 1 process, but put as many threads as possible by your machine
-in this process. We will suppose that the machine has 12 threads per process.
-In this case, run the simulation with:
+As it has been described in the begining of this page supercomputers should be adressed
+with both paradigm.
+ * MPI to go through nodes **and** processors for many socket nodes to handle memory affinity.
+ * OpenMP to minimize imbalance and to manage more efficiently diagnostics at large scale
```

Comment:
I don't understand these two items. IMO, MPI is for balancing between processors. OMP is for balancing inside each processor. Could we keep it simple?

Reply:
Again, this is an open door to go further regarding performances issues.  
In the tuto we avoid, affinity and/or placement issues regarding computing resources, and it could appear simplistic for informed people. It was a way to moderate this simplistic approach.